arch/arm/configs/miyoo_defconfig

```diff
@@ -136,6 +142,7 @@ CONFIG_KEYS=y
 CONFIG_CRYPTO_DRBG_MENU=y
 # CONFIG_CRYPTO_HW is not set
 CONFIG_XZ_DEC=y
+CONFIG_CMA_SIZE_MBYTES=8
```

Comment:
maybe we could just enable CONFIG_CMA and pass different size to kernel via `cma=` arg considering uclibc won't use it?